app/common/constants/keyLocations.js

```diff
@@ -0,0 +1,14 @@
+
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const KeyLocationCodes = {
```

Comment:
nit2: Name it the same you call variables you require to for better grep-ability, so `keyLocations`